template-plugin-auth/frontend/tests/authentication.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { test, expect } from "@playwright/test";
+require("dotenv").config();
```

Comment:
hey @sansx is this the dotenv packaeg I see [here](https://www.npmjs.com/package/dotenv)?

If so, we should add it to `package.json` right?